### DISCUSSION TITLE
Pixmap not working

### DISCUSSION THREAD

**Discussion by rdimeo** [gnudatalanguage/gdl#1506] [Q&A]
Hi. I'm new to GDL but I've been programming IDL for quite some time. I can't get pixmaps to work for double buffering. The following should work fine in IDL but it's not working in GDL. I just get an empty window. Any ideas? Thanks in advance!

Rob

pro test_02
; This simple program is a test for double buffering graphics
device,decomposed = 0 & loadct,0,/silent
xsize = (ysize = 600)
winvis = 0 & winpix = 1
window,winvis,xsize = xsize,ysize = ysize
window,winpix,/pixmap,xsize = xsize, ysize = ysize
wset,winpix
plot,randomn(s,100),psym = 0
wset,winvis
device,copy = [0,0,600,600,0,0,winpix]
wdelete,winpix
end

**Comment by GillesDuvert** [comment]
indeed was a bug: #1509
